### PR TITLE
follow html guidelines

### LINKED ISSUE #12: recommended changes
@go-christianharrel 

Upon code review, I have noticed lines in your code that do not adhere to the [guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p) that are given to us. 

Please fix these issues ASAP.

## First Issue

follow [guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5) in naming classes

https://github.com/PUP-BSIT/exercise-12-vrzn_webdev/blob/68129bc21db3ca10daa2d4f4c02c6dc9cf1b1894/go_christianharrel/go_christianharrel.html#L112

https://github.com/PUP-BSIT/exercise-12-vrzn_webdev/blob/68129bc21db3ca10daa2d4f4c02c6dc9cf1b1894/go_christianharrel/go_christianharrel.html#L201

https://github.com/PUP-BSIT/exercise-12-vrzn_webdev/blob/68129bc21db3ca10daa2d4f4c02c6dc9cf1b1894/go_christianharrel/go_christianharrel.html#L295

make it ```heading-two```

https://github.com/PUP-BSIT/exercise-12-vrzn_webdev/blob/68129bc21db3ca10daa2d4f4c02c6dc9cf1b1894/go_christianharrel/go_christianharrel.html#L116-L146

make it ```box-two```

## Second Issue

Avoid using ```<br>```

https://github.com/PUP-BSIT/exercise-12-vrzn_webdev/blob/68129bc21db3ca10daa2d4f4c02c6dc9cf1b1894/go_christianharrel/go_christianharrel.html#L32

Please make the proper changes and make sure to still adhere to the proper [guidelines](https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p)  that are given to us. 

Thank You!

### PR DESCRIPTION
**Pull request by go-christianharrel**
@JheanKhendrick I have made the necessary adjustments to my code in response to the raised issue and your valuable suggestions. I kindly request a review of these changes. Your feedback is greatly appreciated. **Thank you!**

### CODE REVIEW

**Review by Galope-Jhean — CHANGES_REQUESTED** [review]
I failed to include that you have a container2 class on your HTML please change this as well. 

I apologize for the trouble and I hope you understand

Thank You!

File: go_christianharrel/go_christianharrel.html [hunk 1]
```diff
@@ -109,41 +109,41 @@ <h3>Running</h3>
     </section>
 
     <section class="container container2">
```
Comment (Galope-Jhean) [inline go_christianharrel/go_christianharrel.html]:
```suggestion
    <section class="container container-two">
```

**Review by Galope-Jhean — CHANGES_REQUESTED** [review]

**Review by Galope-Jhean — APPROVED** [review]

File: go_christianharrel/go_christianharrel.html (hunk 1 above)
Comment (go-christianharrel) [reply go_christianharrel/go_christianharrel.html]: I'm having a problem pushing my changes

Comment (Galope-Jhean) [reply go_christianharrel/go_christianharrel.html]: me too

Comment (go-christianharrel) [reply go_christianharrel/go_christianharrel.html]: I can finally push my commit now ^^

**Review by Galope-Jhean — APPROVED** [review]
LGTM!

**Pull request merged**